src/core/resources/transactions/transaction.ts

```diff
@@ -52,6 +51,14 @@ export const fetchTransaction = async ({
   currency: SupportedCurrencyKey;
   chainId: ChainId;
 }) => {
+  if (!SUPPORTED_CHAIN_IDS.includes(chainId)) {
```

Comment:
did you find any case where we pass unsopperted chainids?

all of the instances are using chains coming from useBackendSupportedChains hook like this https://github.com/rainbow-me/browser-extension/blob/master/src/entries/popup/hooks/useTransactionListForPendingTxs.ts#L32 or from that constant https://github.com/rainbow-me/browser-extension/blob/master/src/entries/popup/pages/home/Approvals/Approvals.tsx#L342

Reply:
I removed the `useBackendSupportedChains` hook didn't understand why we had two hooks to fetch txs, replaced with a single `useTransaction` (which from outside this file works the same) to handle the transaction metadata no matter where that data is coming from

Reply:
okok it makes sense